treadmill/cli/__init__.py

```diff
@@ -407,3 +409,75 @@ def echo_yellow(string, *args):
 def echo_red(string, *args):
     """click.echo yellow with support for placeholders, e.g. %s"""
     echo_colour('red', string, *args)
+
+
+def handle_not_authorized(err):
+    """Handle REST NotAuthorizedExceptions"""
+    msg = str(err)
+    msgs = [re.sub(r'failure: ', '    ', line) for line in msg.split(r'\n')]
+    echo_red('Not authorized.')
+    click.echo('\n'.join(msgs), nl=False)
+
+
+def handle_cli_exceptions(exclist):
```

Comment:
@andreikeis Can you have a look at this? I do not want botocore to become a Treadmill core dependency. Is there a better way to wrap up exceptions in your CLI module that TW didn't use?